apps/arena_load_test/lib/arena_load_test/socket_supervisor.ex

```diff
@@ -36,7 +36,6 @@ defmodule ArenaLoadTest.SocketSupervisor do
     end
   end
 
-  def server_host() do
-    System.get_env("SERVER_HOST", "localhost:4000")
-  end
+  def get_server_url("Brazil"), do: "brazil-testing.curseofmirra.com"
+  def get_server_url("Europe"), do: "europe-testing.curseofmirra.com"
```

Comment:
i think we should replace this urls with 
`System.get_env(“BRAZIL_HOST”) and System.get_env(“EUROPE_HOST”)` accordingly